.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ jobs:
   build:
     docker:
       # specify the version you desire here
-      - image: circleci/php:5.6
+      - image: circleci/php:7.1
```

Comment:
This is the one caveat, because https://circleci.com/gh/Rumbleship/php-gateway-client/72